ruby/GETTING_STARTED.md

```diff
@@ -0,0 +1,35 @@
+# Extism
+
+## Getting Started
+
+### Example
+
+```ruby
+require "extism"
+require "json"
+
+Extism.with_context do |ctx|
+  manifest = {
+    :wasm => [{ :path => "../wasm/code.wasm" }],
+  }
+  plugin = ctx.plugin(manifest)
+  res = JSON.parse(plugin.call("count_vowels", "this is a test"))
+  puts res["count"] # => 4
+end
+```
+
+### API
+
+There are two primary classes you need to understand:
+
+* [Context](Extism/Context.html)
+* [Plugin](Extism/Plugin.html)
+
+#### Context
+
+The [Context](Extism/Context.html) can be thought of as a session. You need a context to interact with the Extism runtime. The context holds your plugins and when you free the context, it frees your plugins. We recommend using the [Extism.with_context](Extism.html#with_context-class_method) method to ensure that your plugins are cleaned up. But if you need a long lived context for some reason, you can use the constructor [Extism::Context.new](Extism/Context.html#initialize-instance_method).
```

Comment:
```suggestion
The [Context](Extism/Context.html) can be thought of as a session. You need a context to interact with the Extism runtime. The context holds your plugins and when you free the context, it frees your plugins. We recommend using the [Extism.with_context](Extism.html#with_context-class_method) method to ensure that your plugins are cleaned up. But if you need a long lived context for any reason, you can use the constructor [Extism::Context.new](Extism/Context.html#initialize-instance_method).
```

Reply:
@bhelx - "for some reason" reads to me as if we don't know why a user would want that.